src/main/java/org/jenkinsci/plugins/badge/StatusImage.java

```diff
@@ -94,11 +96,11 @@
      */
     @Override
     public void generateResponse(StaplerRequest req, StaplerResponse rsp, Object node) throws IOException, ServletException {
-        String v = req.getHeader("If-None-Match");
+        /*String v = req.getHeader("If-None-Match");
```

Comment:
I think so. When I was changing the badge length logic, this piece of code was continuing to serve the older version of the badge. That made it super difficult to debug.